bot/cogs/fun.py

```diff
@@ -88,6 +88,15 @@ def __init__(self, bot: Bot):
         if "revive" in message.content.lower():
             await emojify(message, "nou")
 
+    async def on_member_join(self, member):
+        channel = self.bot.get_channel(JOIN_CHANNEL_ID)
+        if member.avatar_url in DEFAULT_AVATARS:
+            text = f'{member.mention} | "*grumbles* <https://support.discordapp.com/hc/en-us/articles/204156688>"'
```

Comment:
We could probably keep the link embed here.

Reply:
Yeah, people are more likely to click the link if they know what it is